### PR TITLE
Added python backend to L0-batcher

### PR DESCRIPTION
**Pull request by fpetrini15**
Addition of python backend testing for L0-batcher

### CODE REVIEW

File: qa/L0_batcher/test.sh [hunk 1]
```diff
@@ -164,14 +184,22 @@ done
 rm -fr var_models && mkdir var_models
 for BACKEND in $BACKENDS; do
     TMP_MODEL_DIR="$DATADIR/qa_variable_model_repository/${BACKEND}_float32_float32_float32"
-
-    for TMP_DIR in $TMP_MODEL_DIR; do
```
Comment (Tabrizian) [inline qa/L0_batcher/test.sh:168]: Having a for loop was not required? Does $TMP_MODEL_DIR always contain a single element?

File: qa/L0_batcher/test.sh [hunk 2]
```diff
@@ -694,4 +722,4 @@ else
     echo -e "\n***\n*** Test FAILED\n***"
 fi
 
-exit $RET
+exit $RET
```
Comment (Tabrizian) [inline qa/L0_batcher/test.sh:725]: Add new line at the end of the file.

**Review by Tabrizian — CHANGES_REQUESTED** [review]

File: qa/L0_batcher/batcher_test.py [hunk 3]
```diff
@@ -222,8 +225,8 @@ def check_status(self, model_name, batch_exec, request_cnt, infer_cnt, exec_coun
                 # Get count from one of the stats
                 self.assertEqual(
                     bc, batch_exec[bs],
-                    "expected model-execution-count {} for batch size {}, got {}".
-                    format(batch_exec[bs], bs, bc))
```
[the remark this answers is not in the document]
Comment (fpetrini15) [reply qa/L0_batcher/batcher_test.py:226]: Yes, I did use the formatter prior to committing. According to the onboarding doc I used version 0.30.0. Is this still the correct version to use?

File: qa/L0_batcher/test.sh (hunk 1 above)
Comment (fpetrini15) [reply qa/L0_batcher/test.sh:168]: It did not seem like this loop was required, each of the backends only had a single entry of the format {BACKEND}_float32_float32_float32.

File: qa/L0_batcher/batcher_test.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Tabrizian) [reply qa/L0_batcher/batcher_test.py:226]: Yes, that is the correct version. Resolving.

File: qa/L0_batcher/test.sh (hunk 2 above)
Comment (fpetrini15) [reply qa/L0_batcher/test.sh]: Should have the newline in the latest commit.

File: qa/L0_batcher/test.sh [hunk 4]
```diff
@@ -164,14 +184,22 @@ done
 rm -fr var_models && mkdir var_models
 for BACKEND in $BACKENDS; do
     TMP_MODEL_DIR="$DATADIR/qa_variable_model_repository/${BACKEND}_float32_float32_float32"
-
-    for TMP_DIR in $TMP_MODEL_DIR; do
-      cp -r $TMP_DIR var_models/. &&
-        (cd var_models/$(basename $TMP_DIR) && \
+    if [ "$BACKEND" == "python" ]; then
```
[the remark this answers is not in the document]
Comment (fpetrini15) [reply qa/L0_batcher/test.sh:187]: This issue should be fixed in the latest commit.

**Review by Tabrizian — APPROVED** [review]
Feel free to merge the PR when the CI passes.

**Pull request merged**